Use of std::setw and std::setprecision requires iomanip header.
ref. https://github.com/ornladios/ADIOS2/issues/4657

author: micketeer@gmail.com
diff -ru testing.orig/adios2/engine/bp/TestBPBufferSize.cpp testing/adios2/engine/bp/TestBPBufferSize.cpp
--- testing.orig/adios2/engine/bp/TestBPBufferSize.cpp	2025-10-10 00:10:20.315482903 +0000
+++ testing/adios2/engine/bp/TestBPBufferSize.cpp	2025-10-10 00:11:55.251095208 +0000
@@ -7,6 +7,7 @@
 
 #include <fstream>
 #include <iostream>
+#include <iomanip>
 #include <stdexcept>
 
 #include <adios2.h>
diff -ru testing.orig/adios2/engine/skeleton/TestSkeletonReader.cpp testing/adios2/engine/skeleton/TestSkeletonReader.cpp
--- testing.orig/adios2/engine/skeleton/TestSkeletonReader.cpp	2025-10-10 00:10:20.318482859 +0000
+++ testing/adios2/engine/skeleton/TestSkeletonReader.cpp	2025-10-10 00:12:08.131906926 +0000
@@ -4,6 +4,7 @@
 #include <iomanip>
 #include <ios>       //std::ios_base::failure
 #include <iostream>  //std::cout
+#include <iomanip>
 #include <stdexcept> //std::invalid_argument std::exception
 #include <vector>
 
diff -ru testing.orig/adios2/interface/TestADIOSSelection.cpp testing/adios2/interface/TestADIOSSelection.cpp
--- testing.orig/adios2/interface/TestADIOSSelection.cpp	2025-10-10 00:06:02.337253830 +0000
+++ testing/adios2/interface/TestADIOSSelection.cpp	2025-10-10 00:11:34.143403744 +0000
@@ -1,4 +1,4 @@
-
+#include <iomanip>
 #include <adios2.h>
 
 #include <gtest/gtest.h>
diff -ru testing.orig/adios2/engine/staging-common/TestCommonRead.cpp testing/adios2/engine/staging-common/TestCommonRead.cpp
--- testing.orig/adios2/engine/staging-common/TestCommonRead.cpp	2025-10-10 00:28:24.310640224 +0000
+++ testing/adios2/engine/staging-common/TestCommonRead.cpp	2025-10-10 00:29:05.110043985 +0000
@@ -7,6 +7,7 @@
 
 #include <chrono>
 #include <iostream>
+#include <iomanip>
 #include <stdexcept>
 #include <thread>
 
diff -ru testing.orig/adios2/engine/staging-common/TestCommonReadAttrs.cpp testing/adios2/engine/staging-common/TestCommonReadAttrs.cpp
--- testing.orig/adios2/engine/staging-common/TestCommonReadAttrs.cpp	2025-10-10 00:28:24.310640224 +0000
+++ testing/adios2/engine/staging-common/TestCommonReadAttrs.cpp	2025-10-10 00:29:20.759815282 +0000
@@ -7,6 +7,7 @@
 
 #include <chrono>
 #include <iostream>
+#include <iomanip>
 #include <stdexcept>
 #include <thread>
 
